package.json

```diff
@@ -36,10 +38,10 @@
     "@fluentui/scheme-utilities": "^8.3.58",
     "@griffel/shadow-dom": "~0.2.0",
     "@nx/devkit": "19.8.14",
+    "@nx/eslint": "19.8.14",
```

Comment:
this changes (no version change) caused by running `nx g @nx/eslint:convert-to-flat-config` 😅